install.js

```diff
@@ -26,11 +29,21 @@ function createTempFile() {
 
 async function downloadChromiumRevision(revision) {
     const tmpPath = await createTempFile();
+    const cacheEntry = cache.match(revision);
+    if (cacheEntry) {
+        debug('Found Chromium archive in cache, skipping download');
+        fs.copyFileSync(cacheEntry, tmpPath);
+
+        return Promise.resolve(tmpPath);
+    }
 
     debug('Downloading Chromium archive from Google CDN');
     const url = utils.getDownloadUrl(revision);
 
-    return _downloadFile(url, tmpPath);
+    return _downloadFile(url, tmpPath).then(destPath => {
```

Comment:
I think it might make sense to rename `desPath` lambda param to `tmpPath` as far as it's actually tmp path.